modules/brittbot/bang.py

```diff
@@ -29,6 +29,12 @@ def rimshot(jenni, input):
 rimshot.rule = "^(!|\x01ACTION )rimshot"
 rimshot.priority = 'medium'
 
+@smart_ignore
```

Comment:
Can you add in a new line above this line? Helps code readability and keeps linters happy.